src/launch/mod.rs

```diff
@@ -261,3 +263,50 @@ pub struct TdxCapabilities {
 
     pub cpuid_configs: Vec<CpuidConfig>,
 }
+
+/// Manually create the wrapper for KVM_MEMORY_ENCRYPT_OP since `kvm_ioctls` doesn't
+/// support `.encrypt_op` for vcpu fds
```

Comment:
Once `kvm_ioctls` adds support for `KVM_MEMORY_ENCRYPT_OP` for vCPUs, we'll remove this. Until then, this will work.